cmd/opfgd/start.go

```diff
@@ -98,11 +96,6 @@ func runStartCmd(ctx client.Context, cmd *cobra.Command, args []string) error {
 		}
 	}()
 
-	// Start finality gadget
-	if err := fg.Startup(fgCtx); err != nil {
-		logger.Fatal("Error starting finality gadget", zap.Error(err))
-	}
```

Comment:
Hmm why do we remove this?

Reply:
The issue with this method is that it start from latest finalized block on l2. As we are testing with anvil latest finalized block is latest block. So the latest block is never being finalized and syncronization can take too long.

The implementation is configurable with:
1. `StartBlockHeight`

2. As well `bsn_activation_height` and `finality_signature_interval` which shoult dictate which block is being checked